lfs/config.go

```diff
@@ -117,6 +117,15 @@ func (c *Configuration) BatchTransfer() bool {
 	return false
 }
 
+func (c *Configuration) PrivateAccess() bool {
+	if v, ok := c.GitConfig("lfs.access"); ok {
+		if v == "private" || v == "PRIVATE" {
```

Comment:
Yeah, I'll fix up nitpicks when things are closer to working. Just wanted to get this linked to your roadmap.